contrib/src/main/java/com/datatorrent/contrib/hbase/HBasePOJOInputOperator.java

```diff
@@ -69,116 +70,105 @@ public Object convertValue( HBaseFieldInfo fieldInfo, Object value)
   public void setup(OperatorContext context)
   {
     try {
-      store.connect();
+      super.setup(context);
       pojoType = Class.forName(pojoTypeName);
       pojoType.newInstance();   //try create new instance to verify the class.
       rowSetter = PojoUtils.createSetter(pojoType, tableInfo.getRowOrIdExpression(), String.class);
-      fieldValueGenerator = FieldValueGenerator.getFieldValueGenerator(pojoType, tableInfo.getFieldsInfo() );
+      fieldValueGenerator = HBaseFieldValueGenerator.getHBaseFieldValueGenerator(pojoType, tableInfo.getFieldsInfo() );
```

Comment:
You could just use the constructor of HBaseFieldValueGenerator... Do you see any advantage is having another public static entry point for generating this object?